pkg/trace/config/config.go

```diff
@@ -448,6 +453,9 @@ type AgentConfig struct {
 	// ContainerProcRoot is the root dir for `proc` info
 	ContainerProcRoot string
 
+	// Azure App Services
+	InAzureAppServices bool
```

Comment:
Are we sure this should be in the Open telemetry code, specifically the OTLP struct? Let's double check and ask Rey if we aren't sure.

Reply:
@purple4reina Is there a better struct to put this variable in or is this appropriate?

Reply:
I agree with @jcstorms1 since this is not related at all to OTLP. Unless you find a better option, I would fallback to `AgentConfig` as the default catch-all.